treeherder/config/settings.py

```diff
@@ -396,6 +396,7 @@ def server_supports_tls(url):
 BUILDAPI_BUILDS4H_URL = "https://secure.pub.build.mozilla.org/builddata/buildjson/builds-4hr.js.gz"
 ALLTHETHINGS_URL = "https://secure.pub.build.mozilla.org/builddata/reports/allthethings.json"
 TASKCLUSTER_TASKGRAPH_URL = 'https://queue.taskcluster.net/v1/task/{task_id}/artifacts/public/full-task-graph.json'
+TASKCLUSTER_INDEX_URL = 'https://index.taskcluster.net/v1/task/'
```

Comment:
`TASKCLUSTER_INDEX_URL` is used in two places, both of which append `'gecko.v2.%s.latest.firefox.decision'`. Perhaps that string should be added to `TASKCLUSTER_INDEX_URL` as well, to avoid the repetition?